findimports.py

```diff
@@ -781,24 +784,22 @@ def printDot(self):
         allNames = set()
         nameDict = {}
         for n, module in enumerate(self.listModules()):
-            module._dot_name = "mod%d" % n
+            module._dot_name = f"mod{n}"
             nameDict[module.modname] = module._dot_name
-            print("  %s[label=\"%s\"];" % (module._dot_name,
-                                           quote(module.label)))
+            print(f"  {module._dot_name}[label=\"{quote(module.label)}\"];")
             allNames |= module.imports
         print("  node[style=dotted];")
         if self.external_dependencies:
             myNames = set(self.modules)
             extNames = list(allNames - myNames)
             extNames.sort()
             for n, name in enumerate(extNames):
-                nameDict[name] = id = "extmod%d" % n
-                print("  %s[label=\"%s\"];" % (id, name))
+                nameDict[name] = id = f"extmod{n}"
+                print(f"  {id}[label=\"{name}\"];")
         for modname, module in sorted(self.modules.items()):
             for other in sorted(module.imports):
                 if other in nameDict:
-                    print("  %s -> %s;" % (nameDict[module.modname],
-                                           nameDict[other]))
+                    print(f"  {nameDict[module.modname]} -> {nameDict[other]};")
```

Comment:
flake8 says
> findimports.py:802:80: E501 line too long (80 > 79 characters)

Reply:
I think I just fixed that too, if I'm not mistaken